src/encoded/static/components/experiment.js

```diff
@@ -137,13 +137,7 @@ const libraryPropertyExtractors = {
     },
 
     strand_specificity: (library) => {
-        const strandSpecificity = library.strand_specificity;
-        let libraryPropertyString = null;
-
-        // A strand_specificity of "false" is still a displayable value.
-        if (strandSpecificity !== undefined) {
-            libraryPropertyString = strandSpecificity ? 'Strand-specific' : 'Non-strand-specific';
-        }
+        const libraryPropertyString = library.strand_specificity;
         return { libraryPropertyString };
     },
```

Comment:
Seems like you can delete this whole property. All it does now is return what it’s been given.

Reply:
Correct, bu out of necessity. The old code used to show each strand per line and the old code served that need. The new way has a different format; it is a summary of all the strands. So, I remove the display logic out. The fact that I have to explain it suggest this part of the library code may need some clean up.

Reply:
Right, you moved the display logic out, but kept the property which effectively does nothing now.